themes/nimma-codes/layouts/index.html

```diff
@@ -7,8 +7,7 @@ <h1>Een cross-language user group voor software developers in de regio Arnhem-Ni
             <p>
                 Er zijn volop softwarebedrijven, webbureaus en designagencies in Arnhem en Nijmegen. Ook zijn er een
                 hogeschool en universiteit met informatica-opleidingen waar mensen worden opgeleid tot programmeur of
-                developer. Toch is er nauwelijks een actieve developer community. Met Nimma Codes willen we daar
-                verandering in aanbrengen.
+                developer. Met Nimma Codes brengen we al deze mensen samen.
```

Comment:
Voor consistentie met de rest van de tekst :)

```suggestion
                developer. Met Nimma.codes brengen we al deze mensen samen.
```